resources/monitoring/templates/grafana/dashboards/etcd.yaml

```diff
@@ -1,32 +1,33 @@
 {{- /*
```

Comment:
I think we can remove all the dashboards in this folder. They are for Kubernetes 1.10 to 1.13, which are not supported any more, right? @a-thaler

Reply:
I vote for it, it turned out to be confusing as they are using old metrics

Reply:
I talked to @a-thaler. He also agrees that we can delete them. @skhalash could you please delete this folder?

Reply:
Sure, will do that

Reply:
Should I also delete the `rules` folder (in favour of `rules-1.14`). It has the same logic and only applies to k8s version < 1.14